examples/two_mass_spring_damper/run.py

```diff
@@ -0,0 +1,53 @@
+import matplotlib.pyplot as plt
+from numpy import linspace
+from pydy.system import System
+from sympy import symbols, Matrix
+from sympy.physics.mechanics import dynamicsymbols, eombase
+
+# Define the dynamic symbols
+
+x1, x2, u1, u2 = dynamicsymbols('x1 x2 u1 u2')
+
+# Define the constant symbols
+
+m1, c1, k1 = symbols('m1 c1 k1')
+m2, c2, k2 = symbols('m2 c2 k2')
+
+# Define the mass matrix and forcing vector
+
+mass_matrix = Matrix([[1, 0,  0,  0],
+                      [0, 1,  0,  0],
+                      [0, 0, m1,  0],
+                      [0, 0,  0, m2]])
+forcing = Matrix([u1,
+                  u2,
+                  -(k1+k2)*x1 + k2*x2 - (c1+c2)*u1 + c2*u2,
+                  k2*x1 - k2*x2 + c2*u1 - c2*u2])
+
+# Form the remaining input lists for the equations of motion class
+
+coordinates = (x1, x2)
+speeds = (u1, u2)
+
+# Initialize the equation of motion class
+
+eom = eombase.EOM(coordinates, speeds, mass_matrix, forcing)
```

Comment:
I'm suggesting that it may be a good idea to have the EOM class contain additional arbitrary equations that are functions of the states, parameters, etc. One example of an equation such as this is an expression for kinetic energy.